config/local_info.xml

```diff
@@ -1,9 +1,80 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<schema>
+<config> 
+        <!-- 
+          ## Begin local override.
+          ## Specify a url attribute to be matched  against the incoming requestor
+          ## url. Only a single url is supported.
+          ## If the urls match, the values specified in this section will override
+          ## the default values given in the default (no url) section below.
+          ## Any valid xml element tree can be specified here but it MUST
+          ## exist in the default section that follows.
+          ## Use of elements 'menus','css' and 'name_mappings' are described
+          ## in the defaults section.
+        -->
+        <local_info url="host-172-16-113-107.nubes.stfc.ac.uk">
+
+          <default_filter_by_scope>true</default_filter_by_scope>
+          
+        </local_info>
+        <!-- 
+          ## End local override
+        -->
+
+        <!-- 
+          ## Begin defaults
+        -->
         <local_info>
-        <!-- If read-only is set as true, the portal will prevent changes to the
+
+          <!-- If read-only is set as true, the portal will prevent changes to the
              data by all users. This can be used for a read-only failover instance. -->
-        <read_only>false</read_only>
+          <read_only>false</read_only>
+
+          <!-- menus
+            ## Give the xml element names from config/web_portal/menu.xml
+            ## to select menu link(s) given in the left-hand menu box.
+            ## Selected menus can be hidden by providing an element value
+            ## of any of hide,false,0 or no. The default behaviour is to 
+            ## show the menu subject to the restrictions encoded in menu.xml
+            ## but any other value but those given above will also work to
+            ## enable the menu display.
+          -->
+          <menus>
+            <Projects>true</Projects>
+            <NGI>true</NGI>
+            <AddNGI>show</AddNGI>
+            <AddProject>show</AddProject>
+            <ServiceGroups>true</ServiceGroups>
+            <AddSite>true</AddSite>
+            <Scopes>true</Scopes>
+            <AdminScopes>true</AdminScopes>
+          </menus>
+          <!-- css
+            ## Provide hex colour values for the background as implemented in
+            ## lib\Gocdb_Services\Config.php (called by 
+            ## docs\web_portal\css\web_portal.php).
+            ## [Note that these values are used as part of the dynamic css
+            ## style sheet fetched by the browser after the base html is
+            ## returned.]
+          -->
+          <css>
+            <backgroundTop>F7F9FE</backgroundTop>
+            <backgroundBottom>D0DFF9</backgroundBottom>
+            <headingTextColour>00AC00</headingTextColour>
+          </css>
```

Comment:
The colours in the `css` block dont seem to be the same as the current colours? (comparing https://172.16.112.125/portal/ to https://gocdb-preprod.egi.eu/portal/)

Reply:
Laziness I'm afraid. Comparing the 'new' web_portal.php with the original plain .css, I used a 2-way linear-gradient only. I guess I thought the difference slight. (screenshot as getting this to render in markdown was beyond my patience.) Could be changed obviously with the addition of more config parameters.

![image](https://user-images.githubusercontent.com/20706670/81323090-c5687580-908c-11ea-8526-db2d2969980c.png)

Reply:
ah I see, could you add the additional config parameters?

Reply:
Updated to allow more generalised colour spec., including diagonal gradient.